frontend/src/pages/Billing/utils/utils.ts

```diff
@@ -77,10 +77,14 @@ export const getMeterAmounts = (
 	const logsMeter = data.billingDetailsForProject?.logsMeter ?? 0
 	const logsQuota =
 		data.billingDetailsForProject?.logsBillingLimit ?? undefined
+	// TODO(vkorolik)
```

Comment:
`TODO`s here and in the file below

Reply:
cleaned this up but leaving some TODOs in place for further changes
* traces pricing logic
* sampling UI settings